scss/_variables.scss

```diff
@@ -352,8 +352,10 @@ $hr-margin-y:                 $spacer !default;
 $table-cell-padding:          .75rem !default;
 $table-cell-padding-sm:       .3rem !default;
 
+$table-color:                 $body-color !default;
```

Comment:
Where do you use this variable?

Reply:
And if it's unused why doesn't fusv catch it :P

Reply:
This was moved up from farther down the file.

Reply:
This looks like a new variable, `$table-color` is not used in `v4-dev`. And indeed, very strange `fusv` doesn't catch this. No idea what's going on here.

Reply:
I think it's used in the sense it's used once, to set the `$table-hover-color` hence why fusv doesn't catch this.

Reply:
Ah I see what you mean now, I assumed it was already in use within `_tables.scss`. We can add it there though.